pkgs/data/themes/pop-gtk/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv
+, fetchurl
+, intltool
+, bash
+, which
+, parallel
+, pkgconfig
+, sassc
+, gtk3
+, inkscape
+, optipng
+, gnome3
+, gtk-engine-murrine
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pop-gtk-theme";
+  version = "3.1.2";
+
+  src = fetchurl {
+    url    = "https://github.com/pop-os/gtk-theme/archive/${version}.tar.gz";
+    sha256 = "15pj5rhhpn3clb87973m9bbmp02z4rv0d88h0idmpdg4i0yj5q8a";
+  };
+
+  nativeBuildInputs = [
+    bash
+    which
+    parallel
+    intltool
+    sassc
+    gtk3
+    inkscape
+    optipng
+  ];
+
+  propagatedUserEnvPkgs = [
+    gnome3.gnome-themes-extra
+    gtk-engine-murrine
+  ];
+
+  preBuild = ''
+    export HOME="$NIX_BUILD_ROOT"
+
+    patchShebangs .
+
+    substituteInPlace Makefile \
+      --replace '/bin/bash' '${bash}/bin/bash'
```

Comment:
I believe this shouldn't be needed as the default builder will invoke make with an appropriate `SHELL`.